classes/class-wc-connect-remote-notices-store.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Remote_Notices_Store' ) ) {
+
+	class WC_Connect_Remote_Notices_Store {
+
+		/**
+		 * @var WC_Connect_API_Client
+		 */
+		protected $api_client;
+
+		protected $allowed_html = array(
+			 'a' => array(
+				 'href' => array(),
+				 'title' => array(),
+			 ),
+			 'br' => array(),
+			 'em' => array(),
+			 'strong' => array(),
+		 );
+
+		public function __construct( WC_Connect_API_Client $api_client ) {
+			$this->api_client = $api_client;
+		}
+
+		public function set_notices( $notices ) {
+			update_option( 'wc_connect_remote_notices', $notices );
+		}
+
+		public function fetch_notices_from_connect_server() {
+			$response = $this->api_client->get_notices( 1, 'en-us' );
+			// TODO: save more than one notice
+			$notices = (object) [
```

Comment:
Thanks! I also see this is the reason the automated tests failed for 5.3 and 5.2. Fixed, rebased, and pushed.